RecoLocalTracker/SiPixelRecHits/interface/SiPixelTemplate.h

```diff
@@ -217,9 +219,9 @@ struct SiPixelTemplateStore { //!< template storage structure
    SiPixelTemplateEntry entx[5][29];  //!< 29 Barrel x templates spanning cluster lengths from -6px (-1.125Rad) to +6px (+1.125Rad) in each of 5 slices [3x29 for fpix]
    void destroy() {};
 #else
-   float* cotbetaY=nullptr;
-   float* cotbetaX=nullptr;
-   float* cotalphaX=nullptr;
+   float* cotbetaY;
```

Comment:
why was the default initialization removed?

Reply:
OK, added it back.